paltas/Configs/acs/config_train_noll_fixz.py

```diff
@@ -0,0 +1,14 @@
+from paltas.Configs.acs.config_train import *
```

Comment:
Same comment as noll config.

Reply:
In fact, it could be could to have a minimal README.rst in this folder that describes the intentions of each config (in one line). That would also resolve my previous comments.